src/Microsoft.Language.Xml/Scanner.cs

```diff
@@ -376,7 +376,7 @@ internal SyntaxToken.Green ScanXmlString(char terminatingChar, char altTerminati
                             var data = MissingToken(null, SyntaxKind.SingleQuoteToken);
                             if (precedingTrivia.Count > 0)
                             {
-                                data = (SyntaxToken.Green)data.WithLeadingTrivia(precedingTrivia.ToListNode());

```

Comment:
I'm torn on whether to allow changes like these as part of this PR. Ideally you would limit this PR to just the changes needed, and leave cleanup into a separate PR if desired. Also probably remove the random whitespace changes (I know the files have mixed line breaks, and it's unfortunate, but let's clean them up later if we want to).